server/models/Notification.ts

```diff
@@ -130,7 +134,7 @@ class Notification extends Model {
   @Column(DataType.UUID)
   teamId: string;
 
-  @AfterSave
+  @AfterCreate
```

Comment:
Why change? This was purposeful to ensure that we don't have events go through the system that haven't been saved to the db.

Reply:
The thinking was that `@AfterSave` also runs on _update_ besides _create_. Going forward _update_ is supposed to be handled by `notificationUpdater`, so keeping `@AfterSave` here would've resulted in a bug?

Reply:
Ahh, you're totally right